ord_schema/interface/search.html

```diff
@@ -0,0 +1,387 @@
+          <a href="#" class="result">{{ reaction_id }}</a><br>
+        {% endfor %}
+        {% if not reaction_ids %}
+          no reactions
+        {% endif %}
+    </div>
+
+    <div id="proto">
+      <pre>
+        no proto
+      </pre>
+    </div>
+
+    <div id="input_template" style="display: none;">
+      <span class="label">input</span>
+      <div class="input edittext"></div>
+      <select class="input">
+        <option value="exact">exact</option>
+        <option value="similar">similar</option>
+        <option value="substructure">substructure</option>
+        <option value="smarts">smarts</option>
+      </select>
+    </div>
+
+    <script>
+
+      // Respond to mouseover on a result to display its pbtxt.
+      function updateProto(reactionId) {
+        const xhr = new XMLHttpRequest();
+        xhr.open('GET', '/id/' + reactionId);
+        xhr.onload = function (event) {
+          $('#proto pre').text(xhr.response);
+        };
+        xhr.send();
+      }
+      
+      // Inputs, output, similarity.
+      function showReagentsTab() {
+        $('#reagents').show();
+        $('#reactions').hide();
+        $('.tab').css('background-color', '');
+        $('#reagentsTab').css('background-color', 'lightblue');
+      }
+      
+      // Reaction IDs, reaction SMILES.
+      function showReactionsTab() {
+        $('#reactions').show();
+        $('#reagents').hide();
+        $('.tab').css('background-color', '');
+        $('#reactionsTab').css('background-color', 'lightblue');
+      }
+
+      // Add an input field, for "+ input" button (and initialization).
+      function addInput() {
+        const input = $('#input_template').clone();
+        const anchor = $('#add_input');
+        input.children().each((index, node) => {
+          anchor.before(node);
+        });
+        // Return value is the text field <div/> of the new input.
+        return anchor.prev().prev();
+      }
+
+      // Get the text from an .edittext div.
+      function getText(node) {
+        return node.text().trim();
+      }
+
+      // Get the matching option for a text field.
+      function getMatchMode(node) {
+        return $('option:selected', node.next()).text()
+      }
+
+      // Get the current value of the similarity slider.
+      function getSimilarity() {
+        return $('#similarity').slider('option', 'value');
+      }
+
+      // Set the value of the similarity slider.
+      function setSimilarity(similarity) {
+        $('#similarity').slider({min: 0, max: 100, value: 100 * similarity});
+      }
+
+      // Read back the user inputs and build a query URL.
+      function exportPredicate() {
+        let path = '/?'
+        let hasReagent = false;
+
+        // Inputs.
+        const inputs = $('.input.edittext');
+        inputs.each((index, node) => {
+          const input = $(node);
+          if (input.is(':hidden')) {
+            // The template.
+            return;
```

Comment:
When the "reactions" tab is shown (so reagents are hidden), this also excludes all inputs in the form -- not just the template.